src/components/Panel/Panel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import RPanel from 'rsuite/lib/Panel'
+
+interface Props {
+  /** The header for the Panel. It can be passed an html element as a header */
+  title?: React.ReactNode
+  /** Determines if the border should be disabled or not. By default false */
+  border?: boolean
+  /** Determines if a shadow should be disabled or not. By default false */
+  shadow?: boolean
+  /** Determines if the Panel can be collapsible. By default false */
+  collapse?: boolean | true
+  /** The value if the Panel should be expanded. By default false */
+  expand?: boolean | true
+  /** The children to render */
+  children?: React.ReactNode
+  /** Function to be run when the Panel is selected. */
+  onSelect?: (event: React.SyntheticEvent<any>) => void
+}
+
+const Panel = (props: Props) => {
+  const { title, border, shadow, collapse, children, expand } = props
+
+  return (
+    <RPanel
```

Comment:
Instead of using the RPanel component, I think that the bootstrap card component has everything that we need. https://react-bootstrap.netlify.com/components/cards/#header-and-footer

Reply:
I'll definitely give it a go. I'm going to be a little busy during the Christmas break so it might take a little longer. Sorry!

Reply:
No worries! If you want help getting it over the finish line, let me know!